packages/plugins/ansible-runner/ansible-runner.spec

```diff
@@ -0,0 +1,110 @@
+%global pypi_name ansible-runner
+
+Name:           %{pypi_name}
+Version:        2.2.1
+Release:        1%{?dist}
+Summary:        A tool and python library to interface with Ansible
+
+License:        ASL 2.0
+URL:            https://github.com/ansible/ansible-runner
+Source0:        https://github.com/ansible/%{pypi_name}/archive/%{version}/%{pypi_name}-%{version}.tar.gz
+
+BuildArch:      noarch
+
+BuildRequires: python%{python3_pkgversion}-devel
+BuildRequires: python%{python3_pkgversion}-setuptools
+BuildRequires: python%{python3_pkgversion}-pbr
```

Comment:
This is the only trouble some build requires from my local testing. Pulpcore packaging carries this so I think we have two options:

 * Carry python-pbr in foreman-packaging
 * Tag over python-pbr from pulpcore-packaging tags into the plugin tags for build time

Reply:
Which python version should this runner be built with? 3.6? 3.8? 3.9?

pulpcore uses 3.9 right now

Reply:
We recently built websockify (https://github.com/theforeman/foreman-packaging/pull/8424). It feels to me that we should use the same Python version in multiple places. I don't strongly care, but it'd be good to reduce the number of Python versions required.
* 3.6: it's the default and others (like EPEL) also build against it, but we don't support EPEL
* 3.8: Ansible (at least in EPEL) is built against it
* 3.9: Katello users have this because of Pulpcore

That doesn't help narrow it down, does it?

Reply:
It does not as the modules are python36, python38 and python39, so parallel usage would be possible. But I would still prefer having the same version so if some dependencies are needed by both, it would not be installed twice for different versions.

Reply:
I know they're parallel installable, but having fewer packages installed is a good thing IMHO.

Reply:
(This is the reason I deleted half of my comment above and just pointed out the version pulpcore uses :hand_over_mouth:)

Ansible uses 3.8 in EL8 (both EPEL and the one in AppStream), so as long we're using that Ansible build, I think we also should use the "matching" Python for the rest of the Ansible ecosystem (part of the hacks we have today are because we're using a Runner on 3.6 with an Ansible on 3.8).

Reply:
Pretty good argument for 3.8. Does that mean we need to start building all our (Python) packages against 3.8? Is this something we configure in our buildroot in Koji?

Reply:
Step 1 for 3.8 -- https://github.com/theforeman/tool_belt/pull/478